public/v2/apps/mongodb-backup-s3.json

```diff
@@ -0,0 +1,134 @@
+                "restart": "always"
+            }
+        },
+        "version": "2"
+    },
+    "instructions": {
+        "start": "Runs mongodump to backup data using cronjob to an s3 bucket",
+        "end": "MongoDB Backup S3 is ready"
+    },
+    "variables": [
+        {
+            "description": "Your aws access key id (for your s3 bucket)",
+            "id": "$$cap_aws_access_key_id",
+            "label": "AWS ACCESS KEY ID",
+            "validRegex": "/^.+$/"
+        },
+        {
+            "description": "Your aws secret access key (for your s3 bucket)",
+            "id": "$$cap_aws_secret_access_key",
+            "label": "AWS SECRET ACCESS KEY",
+            "validRegex": "/^.+$/"
+        },
+        {
+            "description": "Your s3 bucket.",
+            "id": "$$cap_bucket",
+            "label": "BUCKET",
+            "validRegex": "/^.+$/"
+        },
+        {
+            "description": "Name of folder or path to put backups (eg myapp/db_backups/)",
+            "id": "$$cap_backup_folder",
+            "defaultValue": "mongo_dumps/",
+            "label": "BACKUP_FOLDER"
+        },
+        {
+            "description": "The host/ip of your mongodb database",
+            "id": "$$cap_mongodb_host",
+            "label": "MONGODB HOST",
+            "validRegex": "/^.+$/",
+            "defaultValue": "localhost"
+        },
+        {
+            "description": "The port number of your mongodb database",
+            "id": "$$cap_mongodb_port",
+            "label": "MONGODB PORT",
+            "defaultValue": "27017"
+        },
+        {
+            "description": "The username of your mongodb database. If MONGODB_USER is empty while MONGODB_PASS is not, the image will use admin as the default username",
+            "id": "$$cap_mongodb_user",
+            "label": "MONGODB USER",
+            "defaultValue": "admin"
+        },
+        {
+            "description": "The password of your mongodb database",
+            "id": "$$cap_mongodb_pass",
+            "label": "MONGODB PASS",
+            "validRegex": "/^.+$/"
+        },
+        {
+            "description": "The database name to dump. If not specified, it will dump all the databases",
+            "id": "$$cap_mongodb_db",
+            "label": "MONGODB DB"
+        },
+        {
+            "description": "Any extra options to pass to mongodump command. If you see *Unrecognized field 'snapshot'* problem add --forceTableScan here",
+            "id": "$$cap_extra_opts",
+            "label": "EXTRA_OPTS"
+        },
+        {
+            "description": "The interval of cron job to run mongodump. 0 3 * * * by default, which is every day at 03:00hrs.",
+            "id": "$$cap_cron_time",
+            "label": "CRON_TIME",
+            "defaultValue": "0 3 * * *"
+        },
+        {
+            "description": "timezone",
```

Comment:
What's the difference between timezone and cron timezone? Is there any specific reason behind making them two separate fields? Can we not just use one variable for both?

Reply:
It is not clear from the docker image [docs](https://github.com/halvves/mongodb-backup-s3):

```sh
TZ - timezone. default: US/Eastern

CRON_TZ - cron timezone. default: US/Eastern
```

I guess we can use the same variable